recipes/openassetio/all/test_package/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
```

Comment:
Please, remove this file from test_package.

Reply:
Bump https://github.com/conan-io/conan-center-index/pull/15106/files#r1069580708

Reply:
Removed the external fetch of test project and bundled a simplified version instead in abf1234d